SSM/GameManagers/DisguiseManager.java

```diff
@@ -86,6 +86,13 @@ public static void showDisguises(Player player) {
         }
     }
 
+    public static Disguise getDisguise(Player player){
+        if (!disguises.containsKey(player)){
```

Comment:
You don't need to null check hashmaps since they return null if there is no key instead of throwing an exception